engine/src/group.cpp

```diff
@@ -3652,3 +3690,23 @@ void MCGroup::scheduledelete(bool p_is_child)
 		while(t_control != controls);
 	}
 }
+
+// PM-2015-07-02: [[ Bug 13262 ]] Make sure we attach/detach the player when
+//  showing/hiding a group that has a player
+void MCGroup::SetVisibility(MCExecContext &ctxt, uinteger_t part, bool flag, bool visible)
+{
+#ifdef PLATFORM_PLAYER
+    for(MCPlayer *t_player = MCplayers; t_player != nil; t_player = t_player -> getnextplayer())
+    {
+        if (t_player -> getparent() == this)
+        {
+            if (visible)
```

Comment:
visible is the (badly named) parameter determining which property is being set - true -> P_VISIBLE, false -> P_INVISIBLE

The flag param is what the property is being set to.